<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mydoc</class>
 <widget class="QWidget" name="Mydoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文档</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;宋体&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>5</width>
        <height>600</height>
       </size>
      </property>
      <property name="title">
       <string>选择学习文档</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTreeView" name="treeView_docdir">
           <property name="toolTip">
            <string>双击可直接打开选中文档</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;宋体&quot;;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButtonStudy">
             <property name="text">
              <string>学习该文档</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_MystudyRec">
             <property name="text">
              <string>查看学习记录</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_collapse">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>文档名：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_docname">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>当前学习时间：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_time">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: qradialgradient(spread:pad, cx:0.432, cy:0.589545, radius:1.722, fx:0.017, fy:0.0238737, stop:0.0734463 rgba(7, 100, 143, 255), stop:1 rgba(124, 210, 252, 255)); 
color: rgb(31,217,242); </string>
             </property>
             <property name="smallDecimalPoint">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_finishdoc">
             <property name="text">
              <string>关闭文档，完成学习</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>600</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../SimuTrainee.qrc"/>
 </resources>
 <connections/>
</ui>
